traffic_ops/app/lib/API/Cdn.pm

```diff
@@ -181,19 +189,25 @@ sub update {
 		return $self->alert( "a cdn with domain name " . $params->{domainName} . " already exists." );
 	}
 
+	#setting tenant_id to undef if tenant is not set. TODO(nirs): remove when tenancy is no longer optional in the API
+	my $tenant_id = exists($params->{tenantId}) ? $params->{tenantId} :  undef;
```

Comment:
I don't think you need this. This should suffice:

tenant_id => $params->{tenantId}